src/lib/homestore_backend/hs_homeobject.hpp

```diff
@@ -175,10 +175,14 @@ class HSHomeObject : public HomeObjectImpl {
                              pg_.durable_entities().active_blob_count.load(std::memory_order_relaxed));
                 GAUGE_UPDATE(*this, tombstone_blob_count,
                              pg_.durable_entities().tombstone_blob_count.load(std::memory_order_relaxed));
+                GAUGE_UPDATE(*this, total_occupied_space,
+                             pg_.durable_entities().total_occupied_blk_count.load(std::memory_order_relaxed) *
+                                 blk_size);
```

Comment:
When will blk_size be updated?

Reply:
blk_size is a fixed value, it will not be updated after initialized

Reply:
Sorry, I read it wrong.